### ISSUE TITLE
Can't use Where clause in GraphIQL

### ISSUE THREAD

**Issue by DJ-Paxton** [OrchardCMS/OrchardCore#14627]
I'm using Orchard as a Headless CMS and consuming the data from other sites via GraphQL api.
I have a News type with some basic fields and a `List part` of Articles which have some other basic fields.
So far I've been able to fetch all the articles to show a short description and an image and save the `contentItemId`

![image](https://github.com/OrchardCMS/OrchardCore/assets/61999532/a01a5b75-8f14-45c5-870c-23f905030cea)

The thing is when I use that id in a `Where` to show the whole article.

```
{
  newsArticle(
    status: LATEST
    where: {contentItemId: "4pptm00vkzdr223wm2pnwtfa5y"}
  ) {
    shortDescription
    displayText
    tileImage {
      urls
    }
    archive
    contentItemId
  }
}
```

And it will crash with that multi-part identifier error

![image](https://github.com/OrchardCMS/OrchardCore/assets/61999532/59b437e4-b99d-4a0d-a5aa-08a12f3c17ba)

I tried to filter using `Alias` insted of the `contentItemId` but it's the same.

At the moment I'm fetching all the articles, and filtering the JSON response in my client sites but that's just temporary (I hope)
I've seen a similar problem in #3699 and I'm using multitenancy too but I understand that it's solved.

Any ideas? Thanks!

**Comment by DJ-Paxton** [comment]
**Update:**
I found that in the same solution but with the `default Tenant`, the `where` clause works just fine.

And giving a look at the database, I'm seeing that the table names don't match.

![image](https://github.com/OrchardCMS/OrchardCore/assets/61999532/30d93d3b-b162-4d82-ac01-56270bc905e0)

Somehow GraphQl is adding a `_a1` to the `Tenant.TableName` query.

I'll try to replicate the issue in the Core solution to be able to debug it and try to figure it out.
In the meantime if is there any known solution or any ideas it'll be much apreciated.

**Comment by DJ-Paxton** [comment]
**Update2:**
I did not solved the issue yet.
But I found how it works:

First of all, I'm using `SqlServer` and I don't know if this problem exists for other db types.
When creating a new `Tenant` I'm choosing the SAME database to be able to share content types across the platform, and I'm giving a `Table Schema` to separate the tables and here is where this **_a1** is appearing in the GraphQl queries.

Options to work around this:
- Using a different database won't crash the query
- If using a different DB is not an option, instead of giving a `Table Schema` to the Tenant, I found that it'll work correctly by using `Table Prefix` instead.

I hope this helps someone even if the issue is not fixed
